lib/product.rb

```diff
@@ -0,0 +1,20 @@
+class Product
+
+  attr_reader :category, :name, :unit_price, :quantity
+    def initialize(category, name, unit_price, quantity)
+      @category = category
+      @name = name
+      @unit_price = unit_price
+      @quantity = quantity
+    end
+
+    def total_price
+      (@unit_price * quantity.to_i)
+    end
+
+    def hoard
+      if quantity.to_i > 1
+        true
+      end
+    end
```

Comment:
This isn't really what the interaction pattern describes - we wanted a Product do have a default state of `@is_hoarded = false`, with a method `is_hoarded?` that would return the value of the instance variable, and a method `hoard` that would change the value of the instance variable to `true`